lib/order.rb

```diff
@@ -0,0 +1,30 @@
+class Order
+
+  attr_reader :dishes
+
+  def initialize(menu)
+    @dishes = {}
+    @menu = menu
+  end
+
+  def add(dish, quantity)
+    fail NoItemError, "#{dish.capitalize} is not on the menu!" unless menu.has_dish?(dish)
```

Comment:
Metrics/LineLength: Line is too long. [90/80] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)